### PR TITLE
v42 interim chart update

### PR DESCRIPTION
**Pull request by srl295**
~  https://github.com/unicode-org/cldr/commit/060407555310ec78900f7920ac13518a739c824c

### CODE REVIEW

File: docs/charts/42/delta/supplemental-data.html [hunk 1]
```diff
@@ -179,13 +179,22 @@
...
+	<tr><td class='source' rowSpan='16'><a name='Likely' href='#Likely'>Likely</a></td><td class='source' rowSpan='16'><a name='Subtag' href='#Subtag'>Subtag</a></td><td class='source'>ajt</td><td class='target'>▷missing◁</td><td class='target'>ajt_Arab_TN</td></tr>
 	<tr><td class='source'>ann</td><td class='target'>…</td><td class='target'>ann_Latn_NG</td></tr>
 	<tr><td class='source'>oc_ES</td><td class='target'>▷missing◁</td><td class='target'>oc_Latn_ES</td></tr>
 	<tr><td class='source'>pis</td><td class='target'>▷missing◁</td><td class='target'>pis_Latn_SB</td></tr>
 	<tr><td class='source'>smd</td><td class='target'>▷missing◁</td><td class='target'>smd_Latn_AO</td></tr>
 	<tr><td class='source'>snb</td><td class='target'>▷missing◁</td><td class='target'>snb_Latn_MY</td></tr>
 	<tr><td class='source'>tok</td><td class='target'>▷missing◁</td><td class='target'>tok_Latn_001</td></tr>
+	<tr><td class='source'>und_Laoo_AU</td><td class='target'>▷missing◁</td><td class='target'>hnj_Laoo_AU</td></tr>
```
Comment (macchiati) [inline docs/charts/42/delta/supplemental-data.html:189]: This need to be updated, running after the und_Laoo_* removal from the likely subtags.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: docs/charts/42/delta/supplemental-data.html (hunk 1 above)
Comment (srl295) [reply docs/charts/42/delta/supplemental-data.html]: updated

**Comment by macchiati** [comment]
Hmmm. Github thinks that the PR has an ∞ number of files, and hangs when I
try to look at them.

**Comment by srl295** [comment]
Try the full stop “.” Shortcut again

**Comment by macchiati** [comment]
(I forget about that secret option... Even that is still churning away,
trying to load — so I can't view yet.)

What I find very odd is that when I regenerate all the charts after
importing the updated XML data, where essentially every chart changes, I
still don't get so many files that I can't view them in github.

**Comment by srl295** [comment]
Might be file size.

I had a local warning that the .tsv files were over the recommended limit
and to try LFS. Which wouldn’t work for the web hosting I think.

It’s a lot of changes.

I can try again and get the release tag now that there is one.

**Comment by macchiati** [comment]
Maybe, but the file size should not have changed much. I'd expect very few
changes from the last time the charts were generated. The one kicker is
that I converted a bunch of the charts so that the date and version are
included (eg not part of the file), which cuts down drastically on the
number of files each time). I'll try to take a look at your branch locally,
but probably not until Monday.

**Comment by srl295** [comment]
We might consider zipping up the tsv

**Comment by macchiati** [comment]
I don't think the tsvs are a problem, and it is nice to see the changes
over time easily.

**Comment by srl295** [comment]
@macchiati  et all. PTAL /Only one commit

**Review by macchiati — APPROVED** [review]

**Pull request merged**